mautic/mautic
# Mail contacts tab 403 with not 'email:emails:viewothers'

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [>4.4 ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      |  
| Related developer documentation PR URL |  
| Issue(s) addressed                     | 



#### Description:


A user assigned to a role with permission email:emails:viewown and without permission email:emails:viewothers when viewing the contacts tab of an email gets a 403 because the code checks that it has both permissions to generate the contacts grid.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a Role
3. Assign the following permissions:
- Contact Permissions: View Own
- Email Permissions: View Own, Edit Own, Create
4. Create a user and assign the above role
5. With the user created:
- Create a segment and a user 
- Add the segment to the user
- Create a segment email and assign it to the above segment
- Send the email
- User can see the contacts in the contacts tab of the email
